packages/web3-providers/src/NFTScan/apis/TrendingAPI.ts

```diff
@@ -11,12 +11,23 @@ import type { EVM, Response } from '../types/index.js'
 import { fetchFromNFTScan, getContractSymbol } from '../helpers/EVM.js'
 import { LooksRareAPI } from '../../looksrare/index.js'
 import { OpenSeaAPI } from '../../opensea/index.js'
+import { Days } from '../../common.js'
 
 enum NonFungibleMarketplace {
     OpenSea = 'OpenSea',
     LooksRare = 'LooksRare',
 }
 
+// NFTScan will discard range unrecognized range
+const nftscanRangeMap: Record<number, string> = {
```

Comment:
`createLookupTable`

Reply:
OK, but I think it would be a little more complicate